affiliate/initSDKOnPostLoad.js

```diff
@@ -0,0 +1,20 @@
+/*
+Our widgets won't display properly in the editor unless the SDK is run.
+This script is thus for initializing the affiliate SDK in the editor once the page is loaded.
+ */
+const waitForPageLoadAndInitSDK = (oldBlocks, remainingTries) => {
+  if (remainingTries < 1) {
+    return;
+  }
+  const newBlocks = document.querySelectorAll('[data-block]');
+  if (newBlocks.length > 0 && newBlocks.length === oldBlocks.length) {
+    const integrations = document.querySelectorAll('[data-organic-affiliate-integration]');
+    if (integrations.length > 0) {
+      window.empire?.apps?.affiliate?.init?.();
```

Comment:
Actually, SDKv2 will be auto-initialized on itself unless auto-initialization is explicitly disabled
https://github.com/orgnc/platform/blob/81e168049dbdbacb313e9e1831035e690c3941f5/frontend/sdk/src/sdkv2.js#L39

So, instead, to make sure it's correctly initialized and the page processed even if SDKv2 loads after all blocks are loaded we should use `cmd` queue
```js
    window.empire.cmd.push(({ affiliate, core }) => {
        // Core
        if (core && !core.isInitialized()) {
          core.init();
        }

        // Affiliate
        if (affiliate && affiliate?.isEnabled()) {
          if (!affiliate.isInitialized()) {
            affiliate.init();
          } else {
            affiliate.processPage();
          }
        }
    }
```

Reply:
Ah yes, that makes sense. Thanks Sergey.